### PR TITLE
EHSTPCR-1144: Implemented the function get_datalabs_path along with the

### PR DESCRIPTION
**Pull request by esdc-esac-esa-int**
Implemented a function get_datalabs_path to return the complete path of a file in datalabs by combining the datalabs volume path with the path of the file in the table ehst.artifact

### CODE REVIEW

**Comment by esdc-esac-esa-int** [comment]
Build problem with docs/readthedocs.org:astroquery, but this problem seems unrelated to this ticket

File: CHANGES.rst [hunk 1]
```diff
@@ -193,6 +193,10 @@ esa.hubble
 - New methods to retrieve metadata (``get_observations_from_program``) and
   files (``download_files_from_program``) associated to a proposal. [#2910]
 
+- New method ``get_datalabs_path``to return the complete path of a file in
+  datalambs by combining the datalabs volume path with the path of the file
```
Comment (jespinosaar) [inline CHANGES.rst:197]: Please fix the typo here (datalambs)

Comment (esdc-esac-esa-int) [reply CHANGES.rst]: Typo fixed

**Comment by talonsoalbi** [comment]
Fixed the code style issues

**Comment by talonsoalbi** [comment]
Added /<filename> to the output of the function to get the complete path to the file

**Review by bsipocz — APPROVED** [review]
Looks good, thank you!

**Comment by bsipocz** [comment]
@talonsoalbi - if you add the email address you made the commits with to your github profile, these commits will properly show up as your contributions.

**Comment by jespinosaar** [comment]
Many thanks @bsipocz !!

**Pull request merged**